### PR TITLE
VAX-431: Missing relationId

### PR DESCRIPTION
**Pull request by balegas**
Filled relationId on SatTransOp

Also added couple of Int types that I have seen down the wire

### CODE REVIEW

**Review by define-null — CHANGES_REQUESTED** [review]

File: src/satellite/client.ts [hunk 1]
```diff
@@ -279,21 +280,24 @@ export class SatelliteClient extends EventEmitter implements Client {
           txOp = SatTransOp.fromPartial({
             delete: {
               oldRowData: oldRecord,
+              relationId: relation?.id
```
[the remark this answers is not in the document]
Comment (balegas) [reply src/satellite/client.ts]: addressed.

**Comment by balegas** [comment]
Also closes VAX-432

**Review by define-null — APPROVED** [review]
Looks good to me. We need to make sure, we do not close VAX-431, VAX-432 without providing corresponding integration tests in Electric repo.

**Comment by balegas** [comment]
Keeping tickets opens

**Pull request merged**